core/trino-main/src/main/java/io/trino/operator/scalar/json/JsonExistsFunction.java

```diff
@@ -58,8 +59,8 @@
 {
     public static final String JSON_EXISTS_FUNCTION_NAME = "$json_exists";
     private static final MethodHandle METHOD_HANDLE = methodHandle(JsonExistsFunction.class, "jsonExists", FunctionManager.class, Metadata.class, TypeManager.class, Type.class, JsonPathInvocationContext.class, ConnectorSession.class, JsonNode.class, IrJsonPath.class, Block.class, long.class);
-    private static final TrinoException INPUT_ARGUMENT_ERROR = new JsonInputConversionError("malformed input argument to JSON_EXISTS function");
-    private static final TrinoException PATH_PARAMETER_ERROR = new JsonInputConversionError("malformed JSON path parameter to JSON_EXISTS function");
+    private static final Supplier<TrinoException> INPUT_ARGUMENT_ERROR = () -> new JsonInputConversionError("malformed input argument to JSON_EXISTS function");
```

Comment:
There's nothing wrong with having static exceptions. The only thing that will be missing is a proper stack trace pointing at where the error occurred.

However, if you're going to do this, remove the static fields altogether. They are only used in one place.